app/gui/src/presenter/code.rs

```diff
@@ -43,10 +43,10 @@ impl Model {
         }
     }
 
-    fn save_module(&self, content: String) {
+    fn save_module(&self) {
```

Comment:
Again, we do not save a module here, instead store a new save of the entire project.

This method shall be defined and used in the project presenter.